kubernetes/cosmos/values.yaml

```diff
@@ -10,6 +10,7 @@ pfsd:
     pullPolicy: IfNotPresent
 
   service:
+    name: ""
```

Comment:
Prob can be commented out too with a note saying it's a override of whatever default